source/features/view-last-pr-deployment.tsx

```diff
@@ -0,0 +1,27 @@
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import observeElement from '../helpers/simplified-element-observer';
+
+const deploymentSelector = '.js-timeline-item [data-url$="deployed"] .TimelineItem-body .btn[target="_blank"]';
+
+function init(): void {
+	const lastDeployment = select.last(deploymentSelector)!.cloneNode(true);
+	lastDeployment.classList.add('mr-1');
+	lastDeployment.textContent = 'View last deployment';
+
+	select('.gh-header-actions')!.prepend(lastDeployment);
+}
+
+void features.add(__filebasename, {
+	include: [
+		pageDetect.isPRConversation
+	],
+	exclude: [
+		() => !select.exists(deploymentSelector)
+	],
+	init() {
+		observeElement(select('#partial-discussion-header')!.parentElement!, init);
```

Comment:
If a new deployment happens, the button is stale, so maybe it should also include `onNewComments`

Reply:
No need the header will refresh.

I tested it on https://github.com/yakov116/pr-deployments/pull/1